### PR TITLE
Do a git clean when we do a `make ... update`

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 1]
```diff
@@ -310,7 +310,7 @@ help:
 	@echo "Utility targets:"
 	@echo
 	@echo  "- update		Update the project (Git, Node, Pip)"
-	@echo  "- update-git-submodules	Update the Git submodules"
+	@echo  "- update-git-submodules	Update the Git, submodules, and clean unrevised file"
```
Comment (asaunier) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: fileS maybe?
Are the commas really necessary?

Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: better sentence:
`Update Git, the submodules, and clean unrevised files`
for me the meaning is different then `Update the Git submodule ...`!

Comment (asaunier) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: What does "update Git" mean?

Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: `git pull --rebase origin <branch>` but I just see that I'm wrong, then I update the pullrequest

**Comment by asaunier** [comment]
LGTM

**Pull request merged**